var/spack/repos/builtin/packages/py-gcovr/package.py

```diff
@@ -15,9 +15,14 @@ class PyGcovr(PythonPackage):
     homepage = "https://gcovr.com/"
     pypi = "gcovr/gcovr-4.2.tar.gz"
 
+    version("5.2", sha256="217195085ec94346291a87b7b1e6d9cfdeeee562b3e0f9a32b25c9530b3bce8f")
     version("4.2", sha256="5aae34dc81e51600cfecbbbce3c3a80ce3f7548bc0aa1faa4b74ecd18f6fca3f")
 
-    depends_on("python@2.7:,3.5:", type=("build", "run"))
+    depends_on("python@3.7:", when="@5.1:", type=("build", "run"))
+    depends_on("python@3.6:", when="@5.0", type=("build", "run"))
+    depends_on("python@2.7:,3.5:", when="@:4", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@2.7:2,3.5:", when="@:4", type=("build", "run"))
```
`3.0` satisfies `2.7:`

Reply:
Totally missed that, thanks! Fixed.